egs/chime4/s5_1ch/local/run_blstm_gev.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# Copyright 2017 Johns Hopkins University (Author: Aswin Shanmugam Subramanian)
+# Apache 2.0
+
+. ./cmd.sh
+. ./path.sh
+
+# Config:
+nj=10
+cmd=run.pl
+track=6
+. utils/parse_options.sh || exit 1;
+
+if [ $# != 4 ]; then
+   echo "Wrong #arguments ($#, expected 4)"
+   echo "Usage: local/run_blstm_gev.sh [options] <chime4-dir> <chime3-dir> <wav-out-dir> <enhancement-type>"
+   echo "main options (for others, see top of script file)"
+   echo "  --nj <nj>                                # number of parallel jobs"
+   echo "  --cmd <cmd>                              # Command to run in parallel with"
+   echo "  --track <track>            # Chime data to use (1, 2 or 6)"
+   exit 1;
+fi
+
+sdir=$1
+chime3_dir=$2
+odir=$3
+enhancement_type=$4
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+miniconda_dir=$HOME/miniconda3/
+if [ ! -d $miniconda_dir ]; then
+    echo "$miniconda_dir does not exist. Please run '../../../tools/extras/install_miniconda.sh' and '../../../tools/extras/install_chainer.sh';"
+    echo "For chainer install, you have to use cudnn <= 5.1."
+    echo "In the CLSP cluster, $HOME/miniconda3/bin/python -m pip uninstall chainer; . /home/asubraman/.bash_profile_cuda_path; $HOME/miniconda3/bin/python -m pip install chainer==1.16.0 -vvvv --no-cache" && exit 1;
```

Comment:
helpful instructions about installation should be put in the installation scripts themselves.
I looked at the installation scripts and I think they need some work.
They don't do enough work to check dependencies, and I don't feel like they make it easy enough to install dependencies such as CuDNN: you are much too vague and general.  It needs to be more specific, although given the nature of the miniconda installer and the CUDNN download page, it obviously is not going to be possible to completely automate it.  Shinji, can you please work with Aswin on this.

Reply:
@danpovey I have made some changes with the help of Shinji. We have added code to check if cuDNN is present and also to check the version of cuDNN. We have also added few more comments. Please check and let us know your feedback.

Reply:
You can remove these instructions from here: just make sure they are present in the installation scripts.
I don't want you to rely on using that file in your directory for the CLSP cluster: I prefer making people install it themselves, it will help verify that the instructions work.